docs/user/guides/setup/index.rst

```diff
@@ -14,6 +14,11 @@ The following how-to guides help you solve common tasks and challenges in the se
     If you are connecting a Git repository from another provider (for instance Gitea or Codeberg),
     this guide tells you how to add and configure the webhook **manually**.
 
+⏩ :doc:`Adding a configuration file </guides/setup/configuration-file>`
+    When setting up a new project,
```

Comment:
Just for clarity to the user, describe this as all projects, not just new projects:

```suggestion
    To build your documentation project,
```